numcosmo/Makefile.am

```diff
@@ -94,6 +94,7 @@ ncm_sources = \
         math/ncm_stats_dist2d_spline.c       \
 	math/ncm_stats_dist_nd.c             \
 	math/ncm_stats_dist_nd_kde_gauss.c   \
+	math/ncm_stats_dist_nd_kde_studentt.c\
```

Comment:
-In the properties installation part:
I've installed a new property called FREEDOM, to take input from the user for the st_freedom variable in the private struct (referring to the degrees of freedom from the studentt function)

- Need to check the return from _ncm_stats_dist_nd_kde_studentt_eval_m2lnp . I think is +lnnorm, but it is not giving the best results. There are a printf in the end of this function that I've inserted to evaluate some problems, we need to delete it or to comment it out. 

- Check the return from _ncm_stats_dist_nd_kde_studentt_kernel_eval_m2lnp. I didn't change anything from the gauss interpolation file, and that may be a mistake.

-_ncm_stats_dist_nd_kde_studentt_kernel_sample 
I added the gsl_ran_chisq function to the rng file and I'm using it here. I think it is right to put the ncm_vector_scale (y, chi_scale);  where I did. We want all the values of the vector to be scaled with only one value of this function. If I put this scale inside the for loop, after the ncm_vector_set, every single value would be scaled with a different number of this distribution, and I don't know if it is that what we want.

Reply:
Please improve the documentation (you can use ncm_fftlog.c as an example). At the documentation define the free parameter nu and use nu instead of FREEDOM in the code. As a general rule, uppercase names are reserved to macros and enumerators, so, you can use PROP_NU in the enumerator and self->nu in the internal struct. Change the type of nu from uint to gdouble (using uint may result in integer arithmetic and you don't want that).